nbdime/args.py

```diff
@@ -65,7 +66,6 @@ def add_diff_args(parser):
     #parser.add_argument('-d', '--diff-strategy',
     #                    default="default", choices=("foo", "bar"),
     #                    help="specify the diff strategy to use.")
-    pass
```

Comment:
To me `pass` represents "this function intentionally left empty", and keeps linters from complaining about empty functions.